packages/xpetra/src/Utils/ClassList/SC-LO-GO-NO.classList

```diff
@@ -12,9 +12,7 @@
 #Operator
 #ReorderedBlockedCrsMatrix
 TpetraCrsMatrix
-#TpetraMultiVector
```

Comment:
Why did this line change?  Maybe @csiefer2 forgot to modify these in his conversions?

Reply:
This is correct.

They're not SC-LO-GO-NO, they're MULTIVECTOR (because they support SC= ordinal things).